samples/nrf9160/at_client/README.rst

```diff
@@ -0,0 +1,142 @@
+.. _at_client_sample:
+
+nRF9160: AT Client
+##################
+
+The AT Client sample demonstrates the asynchronous serial communication taking place over UART to the nRF9160 modem.
+The sample enables you to use an external computer or MCU to send commands to the LTE-M/NB-IoT modem of your nRF9160 DK.
+
+Overview
+********
+
+The AT Client sample acts as a proxy for sending directives to the nRF9160 modem via AT commands.
+This facilitates the reading of responses or analyzing of events related to the nRF9160 modem.
+The commands can be initiated from a terminal or the `LTE Link Monitor`_, which is an application implemented as part of `nRF Connect for Desktop`_.
+
+For more information on the AT commands, see the `AT Commands Reference Guide`_.
+
+Also see :ref:`modem_info_readme` for information about how an LTE application can interact with the modem.
+
+
+Requirements
+************
+
+* The following development board:
+
+  * nRF9160 DK board (PCA10090)
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf9160/at_client`
+
+.. include:: /includes/build_and_run_nrf9160.txt
+
+
+Connecting to the nRF9160 DK board with LTE Link Monitor
+========================================================
+
+To connect to the nRF9160 DK board with LTE Link Monitor, perform the following steps:
+
+#. Launch the LTE Link Monitor after programming the AT Client sample onto the nRF9160 DK board.
+#. Connect the nRF9160 DK board to the PC with USB.
+#. Power on the nRF9160 DK board.
+#. Click :command:`Select Device` and select the particular board device entry from the drop-down list in the LTE Link Monitor.
+#. Observe that initially the command :command:`AT+CFUN?` is automatically sent to the modem, which returns a value 4, indicating that the modem is in the offline mode.
+#. Observe that the LTE Link Monitor terminal display also shows :command:`AT+CFUN=1` indicating that the modem has changed to the normal mode.
+#. Perform a reset of the nRF9160 DK board by pressing the reset button, which will reboot the board and start the AT Client sample.
+
+
+Testing
+=======
+
+After programming the sample to your board, test the sample by performing the following steps:
+
+#. `Connect to the nRF9160 development board with LTE Link Monitor <Connecting to the nRF9160 DK board with LTE Link Monitor_>`_.
+#. Run the following commands from the LTE Link Monitor terminal:
+
+   * Enter the command :command:`AT+CFUN?`.
+
+     This command connects to an LTE network and triggers the command :command:`AT+CFUN=1` which will change the modem state to the normal mode.
```

Comment:
Updated the sentence. 

Regarding the note, since the setting 'Automatic Requests' influences three different instances in this document, how about adding it as a separate step(step 2)  in "Connecting to the DK board" section ) rather than a note? The "Testing" section references the "Connecting to the DK board" section anyway.

1. Launch the LTE Link Monitor after programming the AT Client sample onto the nRF9160 DK board.
2. Make sure that 'Automatic Setting' is enabled in LTE Link Monitor.
3. Connect the nRF9160 DK board to the PC with USB.
4. Power on the nRF9160 DK board.
..................

Reply:
Added the step.